tests/functional/tester/case_sigquit_remove.go

```diff
@@ -174,6 +174,30 @@ func recover_SIGQUIT_ETCD_AND_REMOVE_DATA(clus *Cluster, idx1 int) error {
 	return err
 }
 
+func inject_SIGKILL(clus *Cluster, index int) error {
+	clus.lg.Info(
+		"disastrous machine failure START",
```

Comment:
A like the drama of the comment... but maybe at least mention that technically it's SIGKILL

Reply:
That's copied from similar `SIGQUIT_ETCD_AND_REMOVE_DATA`, drama was not intentional, was just to lazy to change it :P